app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  has_secure_password
+
+  validates :email, uniqueness: true, presence: true
+  validates_presence_of :password, :role, :active, :address, :city, :state, :zip, require: true
+  validates_numericality_of :role, only_integer: true
```

Comment:
This is good for now, just make a mental note to clamp these once we define what we need, for example if our roles are 0-4 then we'll make a validation to make sure we only have that as inputs.